### PR TITLE
Add release note for 2201.6.3 release

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-6-3
+    - /downloads/swan-lake-release-notes/2201.6.3/
+    - /downloads/swan-lake-release-notes/2201-6-3-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-6-3-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.6.3
+
+<em>Swan Lake 2201.6.3 is the third patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language,
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md:15]:
```suggestion
<em>Swan Lake 2201.6.3 is the third patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language,
```
Do we have any language fixes in addition to the security improvements? If so, we have to add the `Language updates` section mentioning them.

Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md:15]: Fixed in [49ddc](https://github.com/ballerina-platform/ballerina-dev-website/pull/8201/commits/49ddc42a053a7c8dbe08aaea55962ecb901c2311).

File: downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md:15]:
```suggestion
<em>Swan Lake 2201.6.3 is the third patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+and includes a few security improvements</em>
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md:16]:
```suggestion
and a few security improvements</em>.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md (hunk 2 above)
Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md]: Updated in [a48e8](https://github.com/ballerina-platform/ballerina-dev-website/pull/8201/commits/a48e85190b2424b2ed5fe71ddaf0eb1f814ca58c).

File: downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md (hunk 3 above)
Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md]: Updated in [a48e8](https://github.com/ballerina-platform/ballerina-dev-website/pull/8201/commits/a48e85190b2424b2ed5fe71ddaf0eb1f814ca58c).

**Review by sm1990 — APPROVED** [review]

**Pull request merged**